PowerPointLabs/Test/FunctionalTest/SyncLabTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Drawing;
+using System.Windows.Forms;
+using TestInterface;
+using Microsoft.Office.Interop.PowerPoint;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Test.Util;
+using Point = System.Drawing.Point;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using PowerPointLabs;
+
+namespace Test.FunctionalTest
+{
+    [TestClass]
+    public class SyncLabTest : BaseFunctionalTest
+    {
+        private const string UnrotatedRectangle = "Rectangle 3";
+        private const string Oval = "Oval 4";
+        private const string RotatedArrow = "Right Arrow 5";
+        private const string CopyFromShape = "CopyFrom";
+
+
+        protected override string GetTestingSlideName()
+        {
+            return "SyncLab.pptx";
+        }
+
+        [TestMethod]
+        [TestCategory("FT")]
+        public void FT_SyncLabTest()
+        {
+            var syncLab = PplFeatures.SyncLab;
+            syncLab.OpenPane();
+
+            TestCopyDialog(syncLab);
+            TestPasteDialog(syncLab);
+        }
+
+        private void TestCopyDialog(ISyncLabController syncLab)
+        {
+            var actualSlide = PpOperations.SelectSlide(4);
+
+            // no selection
+            MessageBoxUtil.ExpectMessageBoxWillPopUp("", "Please select one item to copy.", syncLab.Copy, "Ok");
+
+            // 2 item selected
+            List<String> shapes = new List<string> { Oval, RotatedArrow };
+            PpOperations.SelectShapes(shapes);
+            MessageBoxUtil.ExpectMessageBoxWillPopUp("", "Please select one item to copy.", syncLab.Copy, "Ok");
+
+            // copy successful
+            PpOperations.SelectShape(CopyFromShape);
+            new Task(() =>
+            {
+                ThreadUtil.WaitFor(1000);
+                SendKeys.SendWait("{ENTER}");
+            }).Start();
+            syncLab.Copy();
```

Comment:
Ok, did you mean this window (screenshot)? I thought it's a msg box, but in fact a customized window. so that's why msgboxutil doesn't work (only works for msgbox as its name indicates)
![capture](https://cloud.githubusercontent.com/assets/3591355/26064347/3f7d5dc8-39c3-11e7-84ca-941a2dd6ca8d.PNG)
so in this case, u'll need to write a controller for it. similar to what u've done for synclab's controller, so like a controller's controller.